### PR TITLE
Drip Infusion

### PR DESCRIPTION
**Pull request by faaaay**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

This is a triple-barrel PR, making three changes to augment cosmetics:

- Shemaghs, sashes and ponchos can now be recoloured in the loadout (precolored sashes and shemaghs are no longer available in the loadout, but they are still in the code as some vendors sell them).
- If your bag is too full to hold all your loadout items, they'll just spawn on the floor rather than vanish into thin air.
- "Cheap melee weapons" have been added to the loadout. Mechanically, these are identical to the machete (except that there's also a generic holster for them and they don't contain plasteel), and come in five flavours: tomahawk, sword, katana, saber, spear. This is because right now, the machete is blatantly better (higher AP, lower size) than the saber/cutlass available in the loadout, which discourages players from taking the melee weapons they actually like. 

	
<hr>
</details>

## Changelog
:cl:
add: added a number of melee weapon reskins to the loadout menu
tweak: if your bag is too full for loadout items, they'll spawn on the floor and not in the void
tweak: Shemaghs, sashes and ponchos taken in the loadout can be recoloured.
removed: you can no longer get sabers, matchies in loadout and you must be a CO/WO/Streward or Preimer to get a real saber
/:cl:

### CODE REVIEW

**Comment by faaaay** [comment]
guh
gimme a minute to fix merge conflicts

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 1]
```diff
@@ -115,6 +115,17 @@
 	path = /obj/item/tool/sword/machete
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:115]: Remove the machete if you dont mind as well

Comment (faaaay) [reply code/modules/client/preference_setup/loadout/lists/armament.dm:115]: sure, gimme a sec

**Comment by faaaay** [comment]
okay, ready to merge

**Pull request merged**